ligo/README.md

```diff
@@ -1,14 +1,43 @@
-# BaseDAO LIGO implementation
+# BaseDAO LIGO
+
+__Note: This is still current in development.__
+
+This is the LIGO implementation of BaseDAO.
+
+The main difference between the two are:
+- `BaseDAO Lorentz`:
+    - uses compile-time configuration to configure various options of baseDAO, which means that each contract that uses `BaseDAO Lorentz` will have its own `michelson` contract.
+
+- `BaseDAO LIGO`:
+    - uses runtime configuration
+    - only one `michelson` contract is used, the user simply defines the configuration as part of originated storage
 
 ## Prerequisites
 
 You will need [ligo executable](https://ligolang.org/docs/intro/installation) installed.
 
 ## Build instructions
 
-Use Makefile to build a contract, for instance
-
+To build the LIGO contract, simply run:
 ```sh
 make all
 ```
 
+## Generating `Storage`
+TODO
+
+## Originating the contract
+To originate `baseDAO LIGO`, run:
+```
+TODO
+```
```

Comment:
Here instead I suggest mentioning `tezos-client` as a possible choice only.

I don't think it's a good idea to write an actual storage in this README, let's only mention an example call.
```suggestion
## Using the contract

You will need [tezos-client](http://tezos.gitlab.io/introduction/howtoget.html) or similar software to originate and interact with the contract on the Tezos network.

For example, if you have compiled the contract and obtained a storage (`<storage>`) as described above you can originate it with:
`tezos-client originate contract myDAO transferring 1 from alice running out/baseDAO.tz --init <storage>`
```